plugins/cwe_type_inference_print/cwe_type_inference_print.ml

```diff
@@ -0,0 +1,14 @@
+open Bap.Std
```

Comment:
As far as I can see, the -d option prints only terms, not attributes of terms. Also, it does not look like the output of -d is meant to be modified by plugins. So I suppose it would be easier to implement our own version of -d. This should not be too hard, but would still wait with that until type inference is feature complete.